source/ed/ed_reader.cpp

```diff
@@ -619,6 +622,138 @@ void EdReader::fill_stop_points(nt::Data& data, pqxx::work& work) {
         }
         data.pt_data->stop_points.push_back(sp);
         this->stop_point_map[const_it["id"].as<idx_t>()] = sp;
+        this->uri_to_idx_stop_point[const_it["uri"].as<std::string>()] = const_it["id"].as<idx_t>();
+    }
+}
+
+void EdReader::fill_access_point_field(navitia::type::AccessPoint* access_point,
+                                       const pqxx::result::iterator const_it,
+                                       const bool from_access_point,
+                                       const std::string& sp_id) {
+    if (!const_it["pathway_mode"].is_null()) {
```

Comment:
very minor suggestion:

```suggestion
    auto is_null_else = [](auto obj, auto f){
    if (obj.is_null()) return;
    f(obj);    
    }; 
    is_null_else(const_it["pathway_mode"], [access_point](auto obj){ access_point->pathway_mode = obj; } );
    is_null_else(const_it["length"], [access_point](auto obj){ access_point->length = obj; }  );
   .......
```